### PR TITLE
feat(ifr): support pixtral

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Describing the newest VLM by Mistral

DRAFT
- [X] Wait for context window
- [X] Wait for supported GPUs
- [X] Test code samples

### CODE REVIEW

File: ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+Pixtral is a vision language model introducing a novel architecture: 12B parameter multimodal decoder plus 400M parameter vision encoder. 
+It can analyze images and offer insights from visual content alongside text. 
+This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension. 
+
+Pixtral is open-weight and distributed under the Apache 2.0 license.
+
+## Why is it useful?
+
+- Pixtral allows to process real world and high resolution images, unlocking capacities such as transcribing handwritten files or payment receipts, extracting information from graphs, captioning images...
```
Comment (nerda-codes) [inline ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx:45]:
```suggestion
- Pixtral allows you to process real world and high resolution images, unlocking capacities such as transcribing handwritten files or payment receipts, extracting information from graphs, captioning images, etc.
```

File: ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+- It offers large context window of up to TBDk tokens, particularly useful for RAG applications.
```
Comment (nerda-codes) [inline ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx:46]:
```suggestion
- It offers large context window of up to TBDk tokens, particularly useful for RAG applications
```

File: ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+</Message>
+
+<Message type="note">
+  Ensure that the `messages` array is properly formatted with roles (system, user, assistant) and content.
+</Message>
+
+### Passing images to Pixtral
+
+1. Image URLs
+If the image is available online, you can just include the image URL in your request as demonstrated above. This approach is simple and doesn't require any encoding.
```
Comment (nerda-codes) [inline ai-data/managed-inference/reference-content/pixtral-12b-2409.mdx:101]:
```suggestion
If the image is available online, you can just include the image URL in your request as demonstrated above. This approach is simple and does not require any encoding.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Comment by tgenaitay** [comment]
Let's go

**Pull request merged**